sarif/src/test/java/com/jetbrains/qodana/sarif/model/MapIgnoreKeysAdapterTest.java

```diff
@@ -0,0 +1,134 @@
+package com.jetbrains.qodana.sarif.model;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.internal.LinkedTreeMap;
+import com.google.gson.reflect.TypeToken;
+import com.jetbrains.qodana.sarif.TestUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.*;
+
+class MapIgnoreKeysAdapterTest {
+
+    private String jsonString;
+    private Map<String, Object> jsonMap;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        jsonString = TestUtils.readStringFromPath("src/test/resources/testData/serializeTest/ignoringKeys.json").replaceAll("\\s", "");
+
+        Map<String, Object> nestedK1 = new LinkedTreeMap<>();
```

Comment:
Why not just read the map with a default gson instance, what's the benefit from building it?

Reply:
True, we don't need to test that gson deserializes correctly I guess